src/cascadia/WindowsTerminal/WindowsTerminal.manifest

```diff
@@ -9,7 +9,7 @@
     <application>
         <!-- Windows 10 1903 -->
         <!-- See https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/xaml-islands -->
-        <maxversiontested Id="10.0.18362.0"/>
+        <maxVersionTested Id="10.0.18362.0"/>

```

Comment:
I'm moderately worried about whether this is case sensitive. I'll check with the manifest team.

Reply:
Great. I'm curious about that one has XML is case sensitive.

Reply:
Wellp, it looks like it _is_ case-sensitive. Somebody owes me a coke, I just haven't figured out who.